okapi-core/src/test/java/org/folio/okapi/ProxyTest.java

```diff
@@ -2010,14 +2010,14 @@ public void testRequestOnly(TestContext context) {
         .post("/_/proxy/tenants/" + okapiTenant + "/install?deploy=true")
         .then().statusCode(200).log().ifValidationFails()
         .body(equalTo("[ {" + LS
-            + "  \"id\" : \"request-only-1.0.0\"," + LS
-            + "  \"action\" : \"disable\"" + LS
-            + "}, {" + LS
             + "  \"id\" : \"request-pre-1.0.0\"," + LS
             + "  \"action\" : \"enable\"" + LS
             + "}, {" + LS
             + "  \"id\" : \"auth-f-module-1\"," + LS
             + "  \"action\" : \"enable\"" + LS
+            + "}, {" + LS
```

Comment:
Matching like this seems a bit prone to break since it relies on whitespace, line breaks and such being just right (at least if I'm understanding how `equalTo` treats it). Wouldn't it be better to parse the object and check its properties?

But please ignore. I can see this kind of matching is used extensively and I can see how addresses the problem of "I want this complex object with these properties". Anyway just curious about the history of this.

Reply:
Yes.. Should use JsonObject .. in a lot of places.. Another PR with just that one day.